go.mod

```diff
@@ -11,16 +11,15 @@ require (
 	github.com/golang/protobuf v1.5.2
 	github.com/google/uuid v1.3.0
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
-	github.com/onosproject/config-models/modelplugin/testdevice-2.0.0 v0.8.8
+	github.com/onosproject/config-models/models/testdevice-2.0.x v0.0.0-20220402084627-ae31ddb22e46
```

Comment:
@ray-milkey I'm not sure how to get the published version of this submodule - if you have any ideas please let me know

Reply:
@ray-milkey can we merge this one?